sphinxcontrib/needs/directives/need.py

```diff
@@ -31,18 +37,23 @@ class NeedDirective(Directive):
                    'hide_tags': directives.flag,
                    'hide_status': directives.flag,
                    'hide_links': directives.flag,
+                   'title_from_content': directives.flag,
```

Comment:
Maybe it shouldn't be a flag.
use_case: needs_title_from_content set in conf.py.
Now there shall be a need with no title but content.
For this need we would need some kind of a boolean option: :title_from_content: false (like the collapse option).
But I'm not sure, if this is  a valid use case...

Reply:
At a minimum in your use case you’d also need needs_title_optional set to True as well or this wouldn’t be allowed. 

For completeness you could do this, but I can’t think of a reason someone would selectively disable a title on an individual requirement if they were using them on all others. 

I don’t mind making the change though.

Reply:
You are right, this would be a realy rare case.
So no change needed here.